lib/ts/recipe/dashboard/recipe.ts

```diff
@@ -167,6 +167,12 @@ export default class Recipe extends RecipeModule {
                 disabled: false,
                 method: "post",
             },
+            {
+                id: USER_API,
+                pathWithoutApiBasePath: new NormalisedURLPath(getApiPathWithDashboardBase(USER_API)),
+                disabled: false,
+                method: "put",
+            },
             {
```

Comment:
how come this is required now? Given that the API is not a new one? This was a bug from before?

Reply:
Yep this was a bug originally. I should have mentioned in the CHANGELOG though so definitely a miss on my part. Updated the CHANGELOG to mention this fix